src/dialects/mariadb/query-generator.js

```diff
@@ -9,6 +12,46 @@ export class MariaDbQueryGenerator extends MySqlQueryGenerator {
     return ['MYSQL', 'INFORMATION_SCHEMA', 'PERFORMANCE_SCHEMA', 'mysql', 'information_schema', 'performance_schema'];
   }
 
+  addColumnQuery(table, key, dataType, options) {
+    options = options || {};
```

Comment:
```suggestion
  addColumnQuery(table, key, dataType, options = {}) {
```

Reply:
@ephys so actually the reason why I did this.

[Add Column](https://github.com/frostzt/sequelize/blob/d41a690a40ecc727e25147e2a93d8e1c11488fe0/src/dialects/abstract/query-interface.js#L452)
[Drop Column](https://github.com/frostzt/sequelize/blob/d41a690a40ecc727e25147e2a93d8e1c11488fe0/src/dialects/abstract/query-interface.js#L468)

In the above abstract function I am explicitly passing `null` incase the options for `ifExists` or `ifNotExists` were not received. This causes an issue with this function since `options` will be set to `null` and then reading the following will fail `options.ifNotExists`. Therefore doing `options = options || {}` sets it an object (otherwise it will fail with `cannot read property of null`).

Reply:
We've actually updated that to undefined on our end https://github.com/sequelize/sequelize/blob/main/src/dialects/abstract/query-interface.js#L462

But apparently we've missed that one https://github.com/sequelize/sequelize/blob/main/src/dialects/abstract/query-interface.js#L478

We don't treat null & undefined as equivalent in this codebase:

```ts
// this won't update anything
Model.update({
  firstName: undefined,
});

// this sets firstName to null
Model.update({
  firstName: null,
});
```

imo it's better if you pass `undefined` instead of `null` and use the default parameter syntax